runbotics/runbotics-orchestrator-ui/src/src-app/components/tables/HistoryTable/HistoryTable.columns.tsx

```diff
@@ -80,12 +83,39 @@ const useProcessInstanceColumns = (): Column[] => {
         },
         {
             Header: translate('Component.HistoryTable.Header.Initiator'),
-            accessor: ({ user, trigger, triggeredBy }) => mapInitiatorLabel({ user, trigger, triggeredBy }),
+            accessor: ({ user, trigger, triggeredBy }) =>
+                mapInitiatorLabel({ user, trigger, triggeredBy }),
+        },
+        {
+            Header: ' ',
+            id: 'button',
+            width: '70px',
+            Cell: ({ row }) =>
+                row.depth === 0 && row.original.input ? (
+                    <BotProcessRunner
+                        process={row.original.process}
+                        externalProcessInstance={row.original}
+                        isProcessInstanceActive={
+                            row.original.status ===
+                                ProcessInstanceStatus.INITIALIZING ||
+                            row.original.status ===
+                                ProcessInstanceStatus.IN_PROGRESS
+                        }
+                        onRunClick={onRerunProcess}
+                    />
+                ) : null,
+            featureKeys: [FeatureKey.PROCESS_START],
         },
     ];
 
+    if (!rerunEnabled) {
+        columns.pop();
+    }
```

Comment:
imagine if some1 adds more columns in the future, unwanted behavior will occur. I suggest to rename id to `rerun` and filter it out depending on prop value

Reply:
btw id button is meaningless

Reply:
id 'button' is used in Table component to make it possible to click on it xd